thirdeye/thirdeye-frontend/app/pods/components/alert-details/template.hbs

```diff
@@ -229,57 +229,4 @@
   {{/unless}}
   <p>{{errorAnomalies}}</p>
   {{yield}}
-
-
-
-
-
-
-
-
-  <!-- {{#if isPendingData}}
```

Comment:
Not required?

Reply:
No. It was reserved by Harley for comparison. We can pull that form git history if needed.